packages/benchmarks/src/quizQuestions/mongodbUniversityAllQuestionBenchmark.ts

```diff
@@ -0,0 +1,67 @@
+import { models } from "../models";
+import "dotenv/config";
+import PromisePool from "@supercharge/promise-pool";
+import { runQuizQuestionEval } from "./QuizQuestionEval";
+import { getQuizQuestionEvalCasesFromBraintrust } from "./getQuizQuestionEvalCasesFromBraintrust";
+import { mongoDbQuizQuestionExamples } from "./mongoDbQuizQuestionExamples";
+import { openAiClientFactory } from "../openAiClients";
+
+async function main() {
```

Comment:
Seems like there's a good amount of repeated structure between this and the badge question benchmark. Fine as separate files for now, but maybe we should look into wrapping this up in a nice abstracted function if we do more of this type of work.

Reply:
yea definitely could do something more abstract. 

they already both wrap `runQuizQuestionEval()` where most of the logic is. 

i think if we keep adding to these, would make sense to abstract further but for now ok as is.